app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Admin::UserController < Admin::ApplicationController
```

Comment:
This was blowing up the 'Users' link for admins.  Needs to be plural to match the file name.